janusgraph-benchmark/src/main/java/org/janusgraph/BenchmarkRunner.java

```diff
@@ -62,17 +49,65 @@ public static void main(String[] args) throws RunnerException, IOException {
             // we sum up results with different params against same benchmark
             metrics.put(benchmark, metrics.getOrDefault(benchmark, 0d) + score);
         }
-        List<Map<String, Object>> outputs = new ArrayList<>();
         for (Map.Entry<String, Double> entry : metrics.entrySet()) {
             outputs.add(new HashMap() {{
                 put("name", entry.getKey());
                 put("value", entry.getValue());
                 put("unit", "ms/op");
             }});
         }
-        String workspace = System.getenv("${GITHUB_WORKSPACE}");
-        File file = new File(workspace, "benchmark.json");
-        System.out.println("writing to " + file.getAbsolutePath());
+    }
+
+    private static void runCqlBenchmarks(ChainedOptionsBuilder cqlBuilder, List<Map<String, Object>> outputs) throws RunnerException, InterruptedException, IOException {
+        CcmBridge.Builder builder =
+            CcmBridge.builder()
+                .withNodes(1);
+        CcmBridge bridge = builder.build();
+        Runtime.getRuntime().addShutdownHook(
+            new Thread(
+                () -> {
+                    bridge.stop();
+                    bridge.remove();
+                }
+            )
+        );
+        bridge.create();
+        bridge.start();
+        System.out.println("Cassandra version = " + bridge.getCassandraVersion());
+
+        cqlBuilder.include("CQL.*Benchmark");
+        transformResults(new Runner(cqlBuilder.build()).run(), outputs);
+    }
+
+    public static void main(String[] args) throws RunnerException, IOException, InterruptedException {
+        final boolean runSpecifiedTests = args.length > 0;
+        final List<Map<String, Object>> outputs = new ArrayList<>();
+        final Supplier<ChainedOptionsBuilder> builderSupplier = () -> new OptionsBuilder()
```

Comment:
Just a stylistic comment: Is there a reason to use the lambda supplier in this case instead of a private method like `getDefaultOptionsBuilder()`. This would shorten the `main` method and thus make it easier to read and easier to extend.